dev/source/docs/editing-the-code-with-atom.rst

```diff
@@ -0,0 +1,16 @@
+.. _editing-the-code-with-atom:
+
+============================
+Editing the code with Atom
+============================
+
+Atom is a free and open-source editor available for Linux, MacOS and Windows developed by GitHub.
+It is a perfect complement to GitHub's primary mission of building better software by working together.
+
+It bundles all tools needed to build ArduPilot : C/C++ support, Python support, Git/GitHub support, :ref:`debugging support <debugging-with-gdb-on-linux>`.
+Using Teletype package, Atom lets developers share their workspace with team members and collaborate on code in real time.
+
+The complete manual is available `here <https://flight-manual.atom.io/>`__.
+To install Atom, follow the instructions available `here <https://flight-manual.atom.io/getting-started/sections/installing-atom/>`__.
+To configure the GitHub package, follow the instructions available `here <https://flight-manual.atom.io/using-atom/sections/github-package/>`__.
+Toc onfigure Teletype package, follow the instructions available `here <https://teletype.atom.io/>`__.
```

Comment:
To configure

Reply:
I prefer Atom because it is more simple and supports GitHub. 
I updated the page as recommended.